lib/iris/fileformats/netcdf/saver.py

```diff
@@ -642,7 +647,10 @@ def write(
             global_attributes = {
                 k: v
                 for k, v in cube_attributes.items()
-                if (k not in local_keys and k.lower() != "conventions")
+                if (
+                    k not in local_keys
+                    and k.lower() not in ["conventions", "iris_extended_grid_mapping"]
```

Comment:
This is unnecessary since it's already in `_UKMO_DATA_ATTRS`, which are included in `local_keys`.